### DISCUSSION TITLE
Windows Key and ability to send combination keys

### DISCUSSION THREAD

**Discussion by viaahmed** [AppGround-io/bluetooth-keyboard-and-mouse-support#275] [Ideas]
I wanted to used this keyboard only to ultimately send keh combinations like [Windows + L]
But I don't see any option to send windows key and combination keys.
If it's already there please any body guide me to it.
If it's not there then please add this feature.

Thanks for the app 😊

**Comment by AdamKane41** [comment]
You can add custom key combinations (and much more) by adding personalized [SHORTCUTS] buttons in the main/default remote screen, or via a custom screen were every buttons are user customizable, powerfull stuff! Peace

**Comment by viaahmed** [comment]
Thanks :)d Now I found the custom layout editor, but could not find
'windows' key anywhere.

Where to find it, could anybody guide me in this?

**Reply by sanctimoniousape** [reply]
1. "Custom layout"
2. Wrench icon in upper right corner.
3. Long-press button to modify, or "Add button".
4. Tap "Win" button, which is the second item in the row starting with "Ctrl" just over halfway down.
5. Press the letter to combine with it (such as E to start Explorer) - both should now be highlighted.
6. Optionally click the three-dot "Menu" icon at the bottom right corner, and select "Rename button" to do just that.
7. Click "Save" in the upper right corner.
